storage/src/mmr/mem.rs

```diff
@@ -1,10 +1,10 @@
 //! A basic MMR where all nodes are stored in-memory.
 
 use crate::mmr::hasher::Hasher;
-use crate::mmr::iterator::{nodes_needing_parents, PathIterator, PeakIterator};
-use crate::mmr::verification::Proof;
+use crate::mmr::iterator::{nodes_needing_parents, PeakIterator};
```

Comment:
nit: you can do `crate::mmr::{}` to avoid these duplicate prefix imports

Reply:
done